controller.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 
-CONTAINERS='["lab", "ctx", "uxo", "tbd", "ipf", "pet", "bit"]'
```

Comment:
Why did you delete this variable?

Reply:
what? i didnt delete anything

Reply:
This red section says otherwise :smile: 

Just add that line back, and I think we'll be good to merge the PR. Thanks!